lms/templates/main.html

```diff
@@ -106,6 +107,10 @@
   google_tag_manager_file = microsite.get_template_path('google_tag_manager.html')
 %>
 
+% if style_overrides_file:
```

Comment:
Nit: I'd prefer that this be immediately after the ``headextra_css`` block so that all the CSS is together. Would that work or does it need to be here?